frontend/packages/knative-plugin/src/models.ts

```diff
@@ -268,6 +268,20 @@ export const EventingTriggerModel: K8sKind = {
   color: knativeEventingColor.value,
 };
 
+export const CamelIntegrationModel: K8sKind = {
+  apiGroup: 'camel.apache.org',
```

Comment:
nit: move this to consts file.

Reply:
had it this as just used in a single place , later move as Const if need to use it more then once